### PR TITLE
fix(receiver): button top line

### PR DESCRIPTION
**Pull request by banklesss**
[YOMO-1145](https://emurgo.atlassian.net/browse/YOMO-1145)

[YOMO-1145]: https://emurgo.atlassian.net/browse/YOMO-1145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/StartMultiTokenTxScreen.tsx [hunk 1]
```diff
@@ -99,7 +101,7 @@ export const StartMultiTokenTxScreen = () => {
           <InputMemo value={memo} onChangeText={handleOnChangeMemo} isValid={!hasMemoError} />
         </ScrollView>
 
-        <Actions>
+        <Actions style={isScrollBarShown && {borderTopWidth: 1, borderTopColor: COLORS.ACTION_GRAY}}>
           <NextButton
```
Comment (SorinC6) [inline apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/StartMultiTokenTxScreen.tsx:113]: @banklesss I think here the right color is `COLORS.BORDER_GRAY`

File: apps/wallet-mobile/src/components/ScrollView/ScrollView.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  return (
+    <RNScrollView
+      onLayout={(event) => {
+        if (!onScrollBarChange) return
+
+        const {height} = event.nativeEvent.layout
+
+        if (wrapperHeight > Math.trunc(height)) onScrollBarChange(true)
+        else onScrollBarChange(false)
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/ScrollView/ScrollView.tsx]:
```
const shouldChange = wrapperHeight > Math.trunc(height);
onScrollBarChange(shouldChange);
props.onLayout?.(event)
```

File: apps/wallet-mobile/src/components/StatusBar.tsx [hunk 3]
```diff
@@ -12,6 +12,8 @@ export const StatusBar = ({type, overrideColor}: Props) => {
   const backgroundColor = type === 'dark' ? COLORS.WHITE : COLORS.BACKGROUND_BLUE
 
   useFocusEffect(() => {
+    if (Platform.OS === 'ios') return
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/StatusBar.tsx]: Commented here https://github.com/Emurgo/yoroi/pull/3061#discussion_r1470065845

Comment (banklesss) [reply apps/wallet-mobile/src/components/StatusBar.tsx]: fixed there

**Review by SorinC6 — APPROVED** [review]

**Review by michalszorad — APPROVED** [review]

**Pull request merged**